src/components/search/_search.scss

```diff
@@ -234,17 +234,17 @@
   }
 
   .#{$prefix}--search--sm .#{$prefix}--search-input {
-    @include typescale('zeta');
+    @include type-style('body-short-01');
     height: rem(32px);
   }
 
   .#{$prefix}--search--lg .#{$prefix}--search-input {
-    @include typescale('epsilon');
     height: rem(40px);
   }
 
   .#{$prefix}--search--xl .#{$prefix}--search-input {
-    @include typescale('epsilon');
+    @include type-style('body-short-02');
+    font-size: carbon--type-scale(4);
```

Comment:
What is `font-size: carbon--type-scale(4);` for and how is it different than the type style?

Reply:
I'm also unsure what this is used for

Reply:
Hi @aagonzales @tw15egan good question - There seems no migration path for `
typescale('epsilon')`, and there seems no `type-style()` corresponding to the font size for large search. That said, `font-size: carbon--type-scale(4)` is for ensuring the font size is the one of large search. Does it make sense to you...?

Reply:
Yes makes sense but why is `@include type-style('body-short-02');` still included then. Are you just overriding the size?

Reply:
> Are you just overriding the size?

Yes, bigger font size, everything else is same as `type-style('body-short-02')`.